packaging/leapp.spec

```diff
@@ -137,6 +140,11 @@ Requires: python-requests
 Requires: python3-six
 Requires: python3-setuptools
 Requires: python3-requests
+%if 0%{?rhel} == 8
+# this prevents situation when someone would like to install leapp with
+# python-3.8+ on RHEL 8, but does not affect any else systems nor upgrades.
+Requires: python(abi) = 3.6
```

Comment:
We should this right away for el9/py3.9

Reply:
@pirat89  Currently we are not creating any builds for el9 yet but probably you are right. we could do that already. give me a minute.

Reply:
Done.